syntax = "proto3";

package util_models;

message Vector2D {
	double x = 1;
	double y = 2;
}